src/main/java/com/connexta/store/service/impl/StoreServiceImpl.java

```diff
@@ -92,6 +92,9 @@ public void addIrm(
     log.info("Successfully indexed datasetId={}", datasetId);
   }
 
+  @Override
+  public void unstage(String datasetId) {}
```

Comment:
✏️ Since this method isn't implemented, could you remove it? Or throw an exception?